data/styles/clear/style-clear/colors.mapcss

```diff
@@ -78,7 +78,9 @@
 @aerodrome5: #C8BCDE;
 
 /* 4.3 Barriers */
-@fence: #999999;
+@barriers: #999999;
+@fencecolor: #000000;
+@citywallcolor: #732f2f;
```

Comment:
we don't add "color" to the color name, let's keep it consistent across the file

Reply:
make sense